RobotsAndPencils/react-gantry
# Webpack setup

So it turns out there's a metric tonne of peer dependencies for these loader setups. The SCSS setup I have found requires a minimum of 3 loaders, pulling in autoprefixer puts that up to 4, and then for production you need another plugin, and then for the resources-loader that puts the final tally at 6.

This setup does not handle SVGs at all. Webpack will error out if you use a .svg somewhere as it has no loaders associated with that file type. There's a slim chance it'll default to the file loader but I am doubtful.

The Readme has a step by step walkthrough of what each loader does and when to the best of my knowledge, as well as links to each loader's readme (some of which leave plenty to be desired).

This completes #11.

## How to Test
Pull this branch and run `npm run start`. It should start a webpack-dev-server that will update when you make changes to either the HomeRoute, ProfileComponent, or any of the Scss files associated with them.

run `npm run build`, it should create a bundle in the `/dist` directory that opens by simply opening the html file within.
